dap_prinz_green_jobs/pipeline/green_measures/industries/sic_mapper/sic_mapper.py

```diff
@@ -204,6 +201,13 @@ def load(self):
             ).items()
         }
 
+        # based on evaluation, we have hard coded for some companies
+        self.hard_coded_sics = {
+            "Menzies Distribution": "49",
+            "Logistics UKs most innovative business": "49",
+            "SaintGobain": "231",
+        }
```

Comment:
I think these could live in `sic_mapper_utils.py` or even another special script - would make it super clear what the hard coding was and where it needed editing (e.g. if more were added)

Reply:
good point!